tensorrt_llm/executor/rpc/rpc_client.py

```
@@ -0,0 +1,497 @@
+                                    future.set_result, response.result)
+                            else:
+                                # Use the original RPCError from the response
+                                target_loop.call_soon_threadsafe(
+                                    future.set_exception, response.error)
+                    self._pending_futures.pop(response.request_id, None)
+
+            except asyncio.CancelledError:
+                # Still handle cancellation for backward compatibility
+                logger_debug("Response reader cancelled")
+                break
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)